content/en/content-management/page-bundles.md

```diff
@@ -131,7 +131,7 @@ A leaf bundle can be made headless by adding below in the front matter
 headless = true
 {{< /code-toggle >}}
 
-There are many use cases of such headless page bundles:
+There are many use cases of such headless page bundles, such as:
```

Comment:
Leave this as is. It is grammatically correct and less verbose.

Reply:
'Many' implies a lot but the docs only provide 2.